sbncode/Calibration/TrackCaloSkimmer_module.cc

```diff
@@ -0,0 +1,460 @@
+////////////////////////////////////////////////////////////////////////
+// Class:       TrackCaloSkimmer
+// Plugin Type: analyzer (art v3_06_03)
+// File:        TrackCaloSkimmer_module.cc
+//
+// Generated at Mon May 17 09:46:34 2021 by Gray Putnam using cetskelgen
+// from cetlib version v3_11_01.
+////////////////////////////////////////////////////////////////////////
+
+#include "TrackCaloSkimmer.h"
+
+#include "art/Utilities/make_tool.h"
+
+// Global functions / data for fitting
+const size_t MAX_N_FIT_DATA = 30;
+
+static int N_FIT_DATA = -1;
+static double FIT_RR[MAX_N_FIT_DATA];
```

Comment:
Use a std::vector? Does this need to be static I guess?

Reply:
When I looked around online this was the suggested way to work with TFitter. I could switch this out for a singleton class in principle, but the fit being performed is simple enough that this works for me.

Reply:
OK, no strong preference from me here then.